app/pathogen/sarscov2/dashboard/(map)/MapAndFilters.tsx

```diff
@@ -98,7 +93,18 @@ export default function MapAndFilters() {
               generatePopupContent={(record) => (
                 <SarsCov2StudyPopupContent record={record} />
               )}
-            />
+              dataPoints={state.filteredData.map((dataPoint) => ({
+                ...dataPoint,
+                longitude: dataPoint.pin_longitude,
+                latitude: dataPoint.pin_latitude,
+                // Convert this to some kind of unique identifier returned from the backend once the SARSCoV2 API returns unique ids.
+                id: `${dataPoint.pin_latitude}-${dataPoint.pin_longitude}-${dataPoint.studyName}`
+              }))} 
+              clusterProperties={{}} 
```

Comment:
placeholders for now but will be implemented if and when we need clustering and we do the work for sero